x-pack/examples/ui_actions_enhanced_examples/public/plugin.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Plugin, CoreSetup, CoreStart } from '../../../../src/core/public';
+import { UiActionsSetup, UiActionsStart } from '../../../../src/plugins/ui_actions/public';
+import { DataPublicPluginSetup, DataPublicPluginStart } from '../../../../src/plugins/data/public';
+import { DrilldownsSetup, DrilldownsStart } from '../../../../x-pack/plugins/drilldowns/public';
```

Comment:
Why we don't reside this example in the `/examples` folder? Do we ship them in dist?

Reply:
> Why we don't reside this example in the /examples folder?

Because this example plugin will have example code that we would like to have X-Pack license.

> Do we ship them in dist?

I don't know.